### PR TITLE
Con 4798 bulk upload empty rows

### PR DESCRIPTION
**Pull request by hkf-tech**

### CODE REVIEW

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.UnitTests/Infrastructure/BulkUploadFileParserTests.cs [hunk 1]
```diff
@@ -139,5 +139,20 @@ public void CorrectNumberOfApprenticeshipMapped()
             var result = _bulkUploadFileParser.GetCsvRecords(_proivderId, _file);
             Assert.AreEqual(2, result.Count());
         }
+
```
Comment (VasanthaKasirajan3008) [inline src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.UnitTests/Infrastructure/BulkUploadFileParserTests.cs:142]: shall we add Test to handle file with empty rows.

**Review by VasanthaKasirajan3008 — APPROVED** [review]
